### ISSUE TITLE
rss feed uses non-standard tags

### ISSUE THREAD

**Issue by anarcat** [cachethq/Cachet#547]
Our RSS feeds looks something like this:

```
<item><title>Previous incidents / incidents précédents</title><message>Previous incidents were only reported on Twitter, see https://twitter.com/hashtag/koumbitstatus?f=realtime&amp;src=hash&#13;
&#13;
----&#13;
&#13;
Les incidents précédents sont seulement signalés sur Twitter, voir https://twitter.com/hashtag/koumbitstatus?f=realtime&amp;src=hash</message><component/><status>Corrigé</status><created_at>2015-03-18 23:13:41</created_at><updated_at>2015-03-18 23:42:39</updated_at></item><item><title>Network reconfiguration / reconfiguration réseau</title><message>Une reconfiguration des routes BGP pour annoncer nos routes directement a été effectuée. Aucune coupure ne s'est produite. Nous avons maintenant une meilleure capacité à annoncer nos propres IPs de différents endroits dans le monde.&#13;
&#13;
----&#13;
&#13;
A network reconfiguration is in process to announce our BGP routes directly. No outage occured. We now have better capabilities at announcing our IPs in different places in the world.</message><component>Réseau</component><status>Corrigé</status><created_at>2015-03-18 23:41:03</created_at><updated_at>2015-03-18 23:42:23</updated_at></item
```

This renders, in Chromium, as this:

```
Previous incidents / incidents précédents


Network reconfiguration / reconfiguration réseau
```

ie. the announcement text body is not included in the feed. I don't believe the `<message>` tag is part of the [RSS standard](http://www.rssboard.org/rss-specification). I would recommend using the `<description>` tag instead.

The tags `<component>` or `<status>` are not standard either for that matter. I would suggest using [`<category>`](http://www.rssboard.org/rss-specification#ltcategorygtSubelementOfLtitemgt) tags instead, for example:

```
<category domain="http://doc.cachethq.io/api/rss/status">Corrigé</category>
<category domain="http://doc.cachethq.io/api/rss/component">Réseau</category>
```

Some review [the output of the RSS validator](http://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fdemo.cachethq.io%2Frss) seems to be in order here. :)

Good news however, the [atom feed seems to be valid](http://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fdemo.cachethq.io%2Fatom). The bad news is that it doesn't have a description either. :)

**Comment by jbrooksuk** [comment]
I've made the RSS feed more compliant.

**Comment by anarcat** [comment]
the feed still doesn't pass the validators, but it looks much better, thanks.

**Comment by jbrooksuk** [comment]
Still a work in progress.

**Comment by jbrooksuk** [comment]
@anarcat in terms of the Feed Validator, do you know why an RSS feed needs to contain information for "Missing atom:link with rel="self""? What's the `atom:link` bit about?

**Comment by jbrooksuk** [comment]
Also this:

> In addition, interoperability with the widest range of feed readers could be improved by implementing the following recommendations.
> line 2, column 582: item should contain a guid element (3 occurrences) [help]

Might not be able to be done yet, because it [prefers a permalink](http://validator.w3.org/feed/docs/warning/MissingGuid.html) to something. We can't link directly to incidents in V1.

**Comment by anarcat** [comment]
i am not sure about atom, but it's a common problem with generated RSS feeds that they need to have an absolute link to the site somehow.

**Comment by jbrooksuk** [comment]
I'm not sure what we'll do here then.

**Comment by slobatch** [comment]
@jbrooksuk 
>Might not be able to be done yet, because it prefers a permalink to something. We can't link directly to incidents in V1.
Perhaps just point to the main public page? It's not a permalink to the specific incident, obviously, but it's perhaps more graceful than not having a url for RSS items at all.

**Comment by jbrooksuk** [comment]
@slobatch that's a good idea.

**Comment by jbrooksuk** [comment]
Moved to second release as we'll need to sort this.

**Comment by jbrooksuk** [comment]
Closed by 9a0d823c9a79a5836c1e8188439484f7902fbb6b
